tests/test_config.py

```diff
@@ -282,6 +282,44 @@ def test_load_config_safe_external(self) -> None:
         SCUBA_YML.write_text(f"image: !from_yaml {external_yml} danger")
         self.__test_load_config_safe(external_yml)
 
+    def test_load_config_image_name(self) -> None:
+        GITLAB_YML.write_text(
+            """
+            two:
+              stage: build
+              image:
+                name: dummian:8.2
+            """
+        )
+        invalid_config(
+            config_text=f"""
+            aliases:
+              two:
+                image:  !from_yaml {GITLAB_YML} two.image
+                script: rip
+            """
+        )
+
+    def test_load_config_gitlab_extensions(self) -> None:
```

Comment:
What are GitLab "extensions"? AFAICT this isn't a term that's used anywhere in YAML, Scuba, or GitLab documentation. If this is testing the `!reference` tag, this should probably have "reference" in the name.